test/functions/vass/return-type-function-failure.prediff

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Is this a link? you can make a symbolic link to util/test/prediff-obscure-module-linenos